config/v1/tests/ingresses.config.openshift.io/SetEIPForNLBIngressController.yaml

```diff
@@ -0,0 +1,224 @@
+    - name: Should allow to set NLB parameters when LBType is NLB.
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Ingress
+        metadata:
+          name: cluster
+        spec:
+          loadBalancer:
+            platform:
+              type: AWS
+              aws:
+                type: NLB
+                networkLoadBalancer:
+                  eipAllocations:
+                  - eipalloc-1234567891011abcd
+                  - eipalloc-1234695789abcdefa
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Ingress
+        metadata:
+          name: cluster
+        spec:
+          loadBalancer:
+            platform:
+                type: AWS
+                aws:
+                  type: NLB
+                  networkLoadBalancer:
+                    eipAllocations:
+                    - eipalloc-1234567891011abcd
+                    - eipalloc-1234695789abcdefa
+    - name: Should not be able to create default ingresscontroller if the value of eipAllocations is not unique.
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Ingress
+        metadata:
+          name: cluster
+        spec:
+          loadBalancer:
+            platform:
+              type: AWS
+              aws:
+                type: NLB
+                networkLoadBalancer:
+                  eipAllocations:
+                  - eipalloc-12345
+                  - eipalloc-12345
+      expectedError: "eipAllocations: Invalid value: \"array\": eipAllocations cannot contain duplicates"
+    - name: Should not be able to create default ingresscontroller if the value of eipAllocations is more than 26 characters.
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Ingress
+        metadata:
+          name: cluster
+        spec:
+          loadBalancer:
+            platform:
+              type: AWS
+              aws:
+                type: NLB
+                networkLoadBalancer:
+                  eipAllocations:
+                  - eipalloc-123450000000000000000000000000000
+      expectedError: "[spec.loadBalancer.platform.aws.networkLoadBalancer.eipAllocations[0]: Too long: may not be longer than 26"
+    - name: Should not be able to create default ingresscontroller if the value of eipAllocations is less than 26 characters.
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Ingress
+        metadata:
+          name: cluster
+        spec:
+          loadBalancer:
+            platform:
+              type: AWS
+              aws:
+                type: NLB
+                networkLoadBalancer:
+                  eipAllocations:
+                  - eipalloc-12345
+      expectedError: "spec.loadBalancer.platform.aws.networkLoadBalancer.eipAllocations[0]: Invalid value: \"eipalloc-12345\": spec.loadBalancer.platform.aws.networkLoadBalancer.eipAllocations[0] in body should be at least 26 chars long"
```

Comment:
What does "in body" mean here?  Is this some weirdness from CEL that you cannot fix in your validation?

Reply:
I just used the error message I saw while running tests. It is not CEL but 
// +kubebuilder:validation:MinLength=26
// +kubebuilder:validation:MaxLength=26